### ISSUE TITLE
Incorrect and missing EXIF values

### ISSUE THREAD

**Issue by humantex** [koken/issues#39]
The EXIF values used for "EXIF.ExposureMode", as of Koken v.0.22.11, are incorrect according to the EXIF v.2.31 specifications. [Full CIPA specs - PDF](http://www.cipa.jp/std/documents/e/DC-008-Translation-2016-E.pdf)  (updated, July 2016) 

From line 1423 in /koken/app/application/models/content.php:
```
'exposure_mode' => array(
	'label' => 'Exposure mode',
	'field' => "EXIF.ExposureMode",
	'values' => array(
		0 => 'Easy shooting',
		1 => 'Program',
		2 => 'Shutter priority',
		3 => 'Aperture priority',
		4 => 'Manual',
		5 => 'A-DEP'
	)
```
These values appear to be a jumble of what should belong to different parts of an array for 'ExposureProgram'. The EXIF spec for ExposureMode only has 3 possible values.

> Tag ID - Tag Name - Write Value - Group
> ++++++++++++++++++++++++++++++++++++
> 0xa402 - ExposureMode - SHORT (int16u) - ExifIFD
> 
> 0 = Auto
> 1 = Manual
> 2 = Auto bracket

Since the file is from Koken's core, any changes made could be overwritten with any updates, so it's not something users would want to monitor and modify every time.

As of the latest version, Koken doesn't include an array with appropriate values for "EXIF.ExposureProgram", which would be:

> Tag ID - Tag Name - Write Value - Group
> ++++++++++++++++++++++++++++++++++++
> 0x8822 - ExposureProgram - SHORT (int16u) - ExifIFD
> 
> 0 = Not defined (Default value)
> 1 = Manual
> 2 = Normal program
> 3 = Aperture priority
> 4 = Shutter priority
> 5 = Creative program (biased toward depth of field / slow speed)
> 6 = Action program (biased toward fast shutter speed / high speed)
> 7 = Portrait mode
> 8 = Landscape mode
> Other = reserved
> 
> 9 = Bulb (Not an accepted value from the spec and ONLY used by Canon for the EOS 7D)

**Comment by humantex** [comment]
An afterthought for 'while you're there' - if anyone plans to commit some changes...

The number of EXIF values exposed by Koken for display are currently limited to 14. Consider expanding them to include things like lens data (make/model, max aperture, etc.), white balance (LightSource), scene type, auto/manual focus data, and a few more of the informative ones. They are all valuable as a learning resource and not just as references to the individual shot's parameters.

**Comment by MikeCostello** [comment]
This will be fixed in v0.22.13, we're currently beta testing.
